packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.types.ts

```diff
@@ -63,24 +61,36 @@ export interface ITooltipHostProps extends React.HTMLAttributes<HTMLDivElement |
    */
   directionalHintForRTL?: DirectionalHint;
 
+  /**
+   * Display property to be applied to the host wrapper div.
+   * When wrapping elements which have display 'inline' or 'inline-block' there are instances when Tooltip covers the target when rendered.
+   * This happens due to default value 'inline' of the wrapper which computes it's line-height from the inherited font-size and being placed
+   * at the base line of the wrapped element. If line-heights of both don't match it will render not in the intended position.
+   * @default 'inline'
+   */
+  display?: IRawStyle['display'];
```

Comment:
I'm concerned about adding a prop for styles elements. I guess I'm not seeing why this can't be handled via styles for the root section? I guess my thought is to apply it in the TooltipHost's default styles function unless there's a reason that won't work?

It seems TooltipHost is still using CSS styling.. should it be converted to JS styling?